Wrappers.h

```diff
@@ -0,0 +1,40 @@
+#include "Building.h"
+#include "Drill.h"
+#include "Elevator.h"
+#include "PencilCase.h"
+#include "SchoolBag.h"
```

Comment:
- don't forget the `#pragma once`
- Don't include what you don't use.  If you aren't using any of these types in this header file, don't include them.  Instead, Forward Declare the types, and then include where you ARE using them: in the cpp file.
- [ ] revise